### ISSUE TITLE
AION Wallet 1.1.0 - Randomly accesses Accounts

### ISSUE THREAD

**Issue by backengineer** [aionnetwork/Desktop-Wallet#186]
I am using Ledger to access accounts.   I initially opened wallet and it defaulted to an account that has a balance of 1500.   On the Accounts Overview - this account is not listed, however, there is another account listed as locked.  

When the locked account on Accounts Overview is unlocked, the account with balance of 1500 is no longer accessible.  

On Opening the wallet a second time (in an attempt to unlock the wallet which became stuck in the middle of a transaction in which I was trying to consolidate accounts) 
The account listed initially listed as locked is now open, and the default account is nowhere to be seen.   

This is becoming a bit of a mess as I am trying to consolidate these two accounts to a SINGLE ACCOUNT that does not disappear randomly.

**Comment by backengineer** [comment]
I am going to turn the above into a feature request...
After restarting the application and paging through 3 pages of public addresses, I found the account buried on the third page of account lists.

REQUEST:  When opening a wallet, display ALL ACCOUNTS WITH BALANCES first in the account list, so that the user does not have to go through 3 pages of accounts to find the account with a balance.

**Comment by centrys-otniel** [comment]
Let me see if I get this right: 
- On your Aion Desktop Wallet (ADW from now on to make things easier) you have created an account that doesn't show up until you click "Unlock account". Let's call this account DesktopAccount1 (DA1)
- Since there is another account listed when the app starts I presume this was imported via a keystore file or a private key. Let's call this DA2
- Having the Ledger connected and Aion app opened on the Ledger device, you selected an address from the first page of the addresses provided by the Ledger. Let's call this LA1.

If I'm correct so far, I have some questions:
1. Which one of these is the account with 1500 AION? DA1, DA2 or LA1

**Comment by backengineer** [comment]
I think there is some confusion regarding terminology...  so to clarify.
I open ADW with Ledger - I only have one account.  DA (as there is no 1, 2, or 3)
I am presented with an initial page of LA's.   
LA1 holds 200 AION.
LA2, LA3, LA4, LA5 hold 0 AION

I hit NEXT to go to the next page of LA's
LA6, LA7, LA8, LA9, LA10 hold 0 AION

I hit NEXT to go to the next page of LA's
LA14 holds 1500 AION
LA11, LA12, LA13, and LA15 hold 0  AION

I have had to page through 3 pages of mostly empty LA's to find the one that contains greatest amount of AION

I am requesting that the Wallet display ALL LA's with NON-ZERO balance FIRST in list.  

REASON:  When I initially converted 1500 AION from MyEtherWallet and transferred to the ADW, I was convinced the total amount had been lost.  Because the first 2 pages of LA's were 0 balance.  It was only until I poked around long enough to keep paging through LA's to find the "lost" funds. 

Crypto is difficult enough for the average user to use.   I am not an average user, I have considerable experience at this, yet I was successfully able to "misplace" 1500 AION because of the multiple pages of LA's.  **User interface design matters**.  Simply sorting by LA with non-zero balance TO THE TOP OF THE LIST will reduce confusion amongst less experienced users of which there are many.
